simd-minimizers/src/lib.rs

```diff
@@ -162,7 +162,7 @@ pub mod private {
 /// Re-export of the `packed-seq` crate.
 pub use packed_seq;
 
```

Comment:
Add a little `## Super-k-mers` paragraph at line 42 above maybe?

And then also a quick hand-made example showing some explicit numbers.